api/src/routing.js

```diff
@@ -415,7 +414,7 @@ app.get('/api/v1/users-info', authorization.handleAuthErrors, authorization.getU
 
 app.postJson('/api/v1/places', function(req, res) {
   auth
-    .check(req, 'can_create_places')
+    .check(req, ['can_edit', 'can_create_places'])
```

Comment:
Change to `hasAllPermissions` (as above).

Reply:
`check` was refactored